### DISCUSSION TITLE
Add Note Level Icon on the Pop-Up Mode!

### DISCUSSION THREAD

**Discussion by ham1204** [vincode-io/Zavala#217] [Support] [answered]
I'm finding Zavala really handy, but there's this one little snag. On the Mac version, I can't add the "Add Note Level" icon to the toolbar during popup edits.

I'd love for the popup behavior to match that of the main window.

**Comment by vincode-io** [comment, marked as answer]
Thanks for letting me know that toolbar button was missing. I didn't intend it to be.

This has been fixed and is in the upcoming 2.3.1 fix release. It is currently in App Review and should be available within the next day or so.

**Reply by ham1204** [reply]
Thanks!
